readme.md

```diff
@@ -499,6 +499,19 @@ An instance of [`CacheableLookup`](https://github.com/szmarczak/cacheable-lookup
 **Note:** This should stay disabled when making requests to internal hostnames such as `localhost`, `database.local` etc.\
 `CacheableLookup` uses `dns.resolver4(..)` and `dns.resolver6(...)` under the hood and fall backs to `dns.lookup(...)` when the first two fail, which may lead to additional delay.
 
+###### dnsIpVersion
+
+Type: `'auto' | 'ipv4' | 'ipv6'`\
+Default: `'auto'`
+
+Indicates which DNS record family to use.\
```

Comment:
Can you maybe include a use-case of when the user would want to override this? It's not clear even to me when I would use this.

Reply:
For example, you can use it to retrieve a user public IPv6
```js
// api6.ipify.org is resolved as IPv4, the request is going to be over IPv4 (the website will respond with your IPv4)
await got('https://api6.ipify.org', {
  dnsIpVersion: 'ipv4'
});

// api6.ipify.org is resolved as IPv6, the request is going to be over IPv6 (the website will respond with your IPv6)
await got('https://api6.ipify.org', {
  dnsIpVersion: 'ipv6'
});
```

Is this example good for the documentation?

There could be also a lot of application specific usages.

Reply:
Yes, that’s a good example.